plugins/woocommerce/lib/composer.json

```
@@ -4,15 +4,15 @@
 	"prefer-stable": true,
 	"minimum-stability": "dev",
 	"require": {
-		"php": ">=7.0",
-		"psr/container": "^1.0"
+		"php": ">=7.2"
 	},
 	"require-dev": {
-		"league/container": "3.3.5"
+		"league/container": "3.3.5",
+	    "psr/container": "^1.1"
```

Real minor, but the indentation is just a little wonky here (should be two tabs, but we have a tab and four spaces):

```suggestion
		"psr/container": "^1.1"
```